Makefile

```diff
@@ -1,5 +1,6 @@
 build: 
-	webpack
+	webpack popup.js public/popup.js
+	webpack background.js public/background.js
 
 package: build
```

Comment:
Can we add the new icons directory to this package command please

`cp -f -a ./{public,icons,popup.html,background.js,styles.css} ./publish`